unlock-app/src/pages/api/auth/[...nextauth].ts

```diff
@@ -3,6 +3,7 @@ import GoogleProvider from 'next-auth/providers/google'
 import { config } from '~/config/app'
 
 export const authOptions = {
+  secret: process.env.NEXTAUTH_SECRET,
```

Comment:
as you know we only "extract" env vars in a single config file. Please do that there and then use config.nextAuthSecret